tests/MetadataTest.php

```diff
@@ -12,7 +12,9 @@
 class MetadataTest extends TestCase
 {
     // SP Metadata entry to request that certain tests be skipped
-    const SkipTestsKey = "SkipTests";    
+    const SkipTestsKey = "SkipTests";
```

Comment:
It's probably not critical, but it may be worth changing these two constants to use ALL_CAPS rather than TitleCase (per our coding conventions).